verticapy/tests_new/pipeline/test_transform.py

```diff
@@ -0,0 +1,581 @@
+        }
+        res = transformation(transform, "public.titanic")
+        assert f'"{name}"' in res.get_columns()
+        drop("public.titanic")
+
+    """
+    Features Transformations test
+    vDataFrame
+    ----------
+    - abs [NOT SUPPORTED]
+    - apply [NOT SUPPORTED]
+    - applymap [NOT SUPPORTED]
+    - polynomial_comb
+    - swap [NOT SUPPORTED]
+
+    vDataColumn
+    -----------
+    - abs
+    - apply
+    - apply_fun
+    - date_part
+    - round
+    - slice
+    """
+
+    @pytest.mark.parametrize(
+        "columns, r",
+        [
+            (["SepalLengthCm", "SepalWidthCm"], 2),
+            (["SepalLengthCm", "SepalWidthCm", "PetalLengthCm"], 3),
+        ],
+    )
+    def test_polynomial_comb(self, columns, r):
+        load_iris()
+        transform = {
+            "new_colm": {
+                # This is a dummy to create the column
+                "sql": "SepalLengthCm",
+                "transform_method": {
+                    "name": "polynomial_comb",
+                    "params": {
+                        "columns": columns,
+                        "r": r,
+                    },
+                },
+            }
+        }
+        res = transformation(transform, "public.iris")
+        elements = columns
+        combinations = itertools.product(elements, repeat=r)
+        unique_combinations = set(
+            tuple(sorted(c, key=elements.index)) for c in combinations
+        )
+        result = ["_".join(perm) for perm in unique_combinations]
+        for col in result:
+            assert f'"{col}"' in res.get_columns()
+        drop("public.iris")
+
+    @pytest.mark.parametrize(
+        "columns",
+        [
+            ("age"),
+            ("fare"),
+            ("survived"),
+        ],
+    )
+    def test_abs(self, columns):
+        load_titanic()
+        transform = {
+            "new_colm": {
+                "sql": columns,
+                "transform_method": {
+                    "name": "abs",
+                },
+            }
+        }
+        res = transformation(transform, "public.titanic")
+        assert '"new_colm"' in res.get_columns()
+        drop("public.titanic")
+
+    @pytest.mark.parametrize(
+        "columns, func, copy_name",
+        [
+            (["age"], "POWER({},2)", None),
+            (["age"], "POWER({},2)", "age_pow2"),
+        ],
+    )
+    def test_apply(self, columns, func, copy_name):
+        load_titanic()
+        transform = {
+            "new_colm": {
+                "sql": columns[0],
+                "transform_method": {
+                    "name": "apply",
+                    "params": {
+                        "func": func,
+                        "copy_name": copy_name,
+                    },
+                },
+            }
+        }
+        res = transformation(transform, "public.titanic")
+        assert '"new_colm"' in res.get_columns()
```

Comment:
shouldn't we also be testing the result of the apply function along with checking the name?